src/Plugin/Action/Order/ExportOrderAction.php

```diff
@@ -33,7 +34,13 @@ public function handle(array $parameters): Response
     {
         $orders = $this->orderRepository->getMany($parameters['orderIds']);
 
-        $shipments = $orders->generateShipments();
+        $shipments = $orders->generateShipments()
```

Comment:
ik ben het hier niet echt mee eens, dit is de enige plek waar generateShipments gebruikt wordt, en ik zou dan ook eigenlijk verwachten dat die alleen de nieuwe shipments teruggeeft